.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,47 @@
+name: Continuous Integration
+
+on: [push, pull_request]
+
+jobs:
+  validate:
+    name: Validate Impletementation Guide
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        node-version: [10, 12]
+    steps:
+      - name: Check out branch
+        uses: actions/checkout@v2
+
+      - name: Set up OpenJDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+
+      - name: Set up Jekyll
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install ruby-full build-essential zlib1g-dev
+          sudo gem install jekyll bundler
+
+      - name: Set up Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install SUSHI and add FSH definitions
+        run: |
+          npm install -g fsh-sushi
+          sushi .
+
+      - name: Update IG publisher
+        run: |
+          sudo apt-get update
```

Comment:
not that it matters, but update was already run in the jekyll step

Reply:
Removed!